app/models/manageiq/providers/redhat/infra_manager/inventory/strategies/v4.rb

```diff
@@ -74,6 +74,7 @@ def refresh
         preloaded_disks = collect_disks_as_hash(res[:disk])
         collect_disks_from_attachments(res[:vm], preloaded_disks)
         collect_disks_from_attachments(res[:template], preloaded_disks)
+        res[:vnic_profiles] = collect_vnic_profiles
```

Comment:
Why not to stay consistent with the code? Would be better to have `collect_vnic_profiles(res[:vnic_profiles])`